src/app/services/journey.service.ts

```diff
@@ -40,30 +40,56 @@ export class JourneyService {
     return destination$;
   }
 
+  /**
+   * search by destination
+   * @param journey to update
+   */
+  updateJourney(journey: Journey): Observable<Array<Journey>> {
+      const source$ = this.apollo.mutate({
+          mutation: gql`
+              mutation update{
+                  updateJourney(input:{ id: "${journey.id}" destination: "${journey.destination}" }) {
+                      id
+                      destination
```

Comment:
on the form there is the price, so it will never be updated as the mutation does not contain it ?